### PR TITLE
IBX-3767: Document how to use fastly CLI on post deploy hook

### PR DESCRIPTION
**Pull request by Nattfarinn**
| Question                                  | Answer
| ---------------------------------------- | ------------------
| **JIRA issue**                          | Ref. to: https://issues.ibexa.co/browse/IBX-3767
| **Type**                                   | improvement
| **Target Ibexa version** | `v4.3`
| **BC breaks**                          | no

#### Checklist:
- [x] Checked that target branch is set correctly (`main` for features, the oldest supported for bugs).
- [x] Asked for a review (ping `@ibexa/engineering`).

### CODE REVIEW

**Comment by adamwojs** [comment]
You can rebase PR @Nattfarinn.  https://github.com/ibexa/post-install/pull/58 has been merged.

File: resources/platformsh/ibexa-content/4.3.x-dev/.platform.app.yaml [hunk 1]
```diff
@@ -210,6 +210,11 @@ hooks:
         # Depending on your VCL, buy default this would trigger a soft purge (expiry) and allow grace period, however
         # even so if your change only affected certain subset of content, ideally you should only clear specific tags:
         ##bin/console fos:httpcache:invalidate:tag l44 c33 ct2 ...
+        
+        # It is possible to manipulate your Fastly VCL configuration directly from command line using Fastly CLI. 
+        # Official documentation on how to install: https://developer.fastly.com/learning/tools/cli#installing 
+        # Example VCL snippet upload using vcl_recv hook (remember about FASTLY_SERVICE_ID and FASTLY_KEY environmental variables):
+        ##fastly vcl snippet create --name="Enable Fastly IO" --version=active --autoclone --type recv --content=vendor/ibexa/fastly/fastly/ibexa_image_optimizer.vcl
```
Comment (adamwojs) [inline resources/platformsh/ibexa-content/4.3.x-dev/.platform.app.yaml:217]: As discussed on private: we need example of how to upload main VCL file as well

**Review by adamwojs — APPROVED** [review]

**Review by ViniTou — APPROVED** [review]

**Review by webhdx — APPROVED** [review]

**Review by konradoboza — APPROVED** [review]

**Review by vidarl — COMMENTED** [review]
Is is really desirable to put this kind of documentation in the .platform.app.yaml file ?
On Ibexa Cloud, p.sh will (are at least supposed to ) install the .vcl and snippets during onboarding. p.sh want to avoid doing any further updates it after that, so it is good that we have some documentation on how to use Fastly CLI though.

I think the better place is doc.ibexa.co. I have this PR pending btw to improve Fastly documentation, but it does not address Fastly CLI : https://github.com/ezsystems/developer-documentation/pull/1848/files.
Not all Fastly users are necessarily on Ibexa Cloud, so I think it is important to not have the Fastly doc only in p.sh context

I don't think it hurts to have it both here and on doc.ibexa.co though, but there are then two places to maintain in regards to changes.

For instance, when Fastly-IO will be supported shortly

@DominikaK

File: resources/platformsh/ibexa-content/4.3.x-dev/.platform.app.yaml [hunk 2]
```diff
@@ -210,6 +210,12 @@ hooks:
         # Depending on your VCL, buy default this would trigger a soft purge (expiry) and allow grace period, however
         # even so if your change only affected certain subset of content, ideally you should only clear specific tags:
         ##bin/console fos:httpcache:invalidate:tag l44 c33 ct2 ...
+        
+        # It is possible to manipulate your Fastly VCL configuration directly from command line using Fastly CLI. 
+        # Official documentation on how to install: https://developer.fastly.com/learning/tools/cli#installing 
+        # Example VCL snippet upload using vcl_recv hook (remember about FASTLY_SERVICE_ID and FASTLY_KEY environmental variables):
+        ##fastly vcl custom create --name="Ibexa VCL" --main --version=latest --autoclone  --content=vendor/ibexa/fastly/fastly/ez_main.vcl
+        ##fastly vcl snippet create --name="Shielding" --version=active --autoclone --type recv --content=vendor/ibexa/fastly/fastly/snippet_re_enable_shielding.vcl
```
[the remark this answers is not in the document]
Comment (Nattfarinn) [reply resources/platformsh/ibexa-content/4.3.x-dev/.platform.app.yaml:218]: This is just example of how to use Fastly CLI in general, not how to enable and configure Fastly on our side. :)

[the remark this answers is not in the document]
Comment (lserwatka) [reply resources/platformsh/ibexa-content/4.3.x-dev/.platform.app.yaml:218]: Can we get rid of ez prefix?

**Pull request merged**